### DISCUSSION TITLE
Does tweepy support "like" event from twitter?

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#1054] [Questions] [answered]
Tried "liking" someone else's post on my timeline but no event seems to get triggered by the StreamListener.  Regular posts and retweets get triggered however.

**Comment by Harmon758** [comment, marked as answer]
This is no longer applicable now, since User and Site Streams have been deprecated and retired in favor of the Account Activity API.
For Account Activity API support, see #1075.

See also:
https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/migration/us-ss-migration-guide.html

https://blog.twitter.com/developer/en_us/topics/tools/2018/enabling-all-developers-to-build-on-the-account-activity-api.html
https://blog.twitter.com/developer/en_us/topics/tools/2017/announcing-more-functionality-to-improve-customer-engagements-on-twitter.html
https://blog.twitter.com/developer/en_us/topics/tools/2017/building-the-future-of-the-twitter-api-platform.html

https://twittercommunity.com/t/details-and-what-to-expect-from-the-api-deprecations-this-week-on-august-16-2018/110746
https://twittercommunity.com/t/reminder-site-streams-user-streams-and-legacy-dm-endpoints-will-be-sunset-august-16th/109482
https://twittercommunity.com/t/building-on-site-streams-user-streams-or-direct-messages-action-required-to-preserve-and-prevent-disruption-to-your-twitter-integration/105736
https://twittercommunity.com/t/date-for-full-deprecation-of-user-streams-site-streams-and-legacy-direct-message-api-endpoints/98734
